### DISCUSSION TITLE
Powered USB 3.0 Hub (Otolla)

### DISCUSSION THREAD

**Discussion by labberdabber** [SmoothWAN/SmoothWAN#52] [General & Help]
Hi! I absolutely love this project! Thank you for all your great work on this!  I am attempting to migrate away from just running Speedify directly off the RPI4 with a powered USB 3.0 hub with a tethered Android and M2000 Hotspot.  All works great with Rasp-OS/Speedify, but when I move to SmoothWan,  the devices are only discoverable by OpenWRT when directly connected to the RPi4.  If I introduce the USB hub, they are MIA.  Did not think there was a specific driver needed for the powered hub, but curious if anyone else has run into this issue.

**Comment by TalalMash** [comment]
Hello, thanks

Yes the ports outside the Pi are not automatically setup as hubs differ greatly, go to Network -> Interfaces -> Devices:
![image](https://user-images.githubusercontent.com/96490382/183262584-2611547e-d315-4487-8bba-3fd35e10972c.png)
You should see your device name there
Then switch to Interfaces tab and click "Add new interface"
![image](https://user-images.githubusercontent.com/96490382/183262614-fccdb71b-9017-4d1c-ab23-4053dbad4f81.png)
![image](https://user-images.githubusercontent.com/96490382/183262620-ade01582-7a51-47f1-93c5-c3d9dabca0e6.png)
Switch to Firewall tab and select the red zone
![image](https://user-images.githubusercontent.com/96490382/183262631-3f33d159-9a99-4ba5-94b8-9ed2b14f3e81.png)
Click Save - Save & Apply
It will show on Speedify, you won't be able to rename the interfaces though.

**Reply by labberdabber** [reply]
Thank you!! That worked perfectly!
